templates/components/material_footer.html

```diff
@@ -10,7 +10,7 @@ <h5 class="white-text">Useful Links</h5>
                 <ul>
                     <li><a class="white-text" target="_blank" href="http://biorxiv.org/content/early/2016/10/31/084582">Publication</a></li>
                     <li><a class="white-text" href="/faq">Frequently Asked Questions</a></li>
-                    <li><a class="white-text" href="#team">The Team</a></li>
+                    <li><a class="white-text" href="about#team">The Team</a></li>
```

Comment:
it's usually better practice to have a leading `/`...

See (this StackOverflow)[http://stackoverflow.com/questions/10659459/starting-with-a-forward-slash-in-html-for-href] for why...

It shouldn't make much difference at the moment (based on the current structure of the website) but it would better to fix 👍